chore: Update README.md (#44)
linuxdeepin/deepin-compressor: README.md

- make
- make install

## Documentations

- [Development Documentation](https://linuxdeepin.github.io/)

>  - [User Documentation](https://wikidev.uniontech.com/index.php?title=%E9%9F%B3%E4%B9%90) | [用户文档](https://wikidev.uniontech.com/index.php?title=音乐)

Reviewer: ```suggestion
```

## Getting help

- [Official Forum](https://bbs.deepin.org/) for generic discussion and help.
- [Developer Center](https://github.com/linuxdeepin/developer-center) for BUG report and suggestions.
- [Wiki](https://wiki.deepin.org/)

## Getting involved

We encourage you to report issues and contribute changes

- [Contribution guide for developers](https://github.com/linuxdeepin/developer-center/wiki/Contribution-Guidelines-for-Developers-en) (English)
- [开发者代码贡献指南](https://github.com/linuxdeepin/developer-center/wiki/Contribution-Guidelines-for-Developers) (中文)

> - [Translate for your language on Transifex](#) *please update to the actual Transifex link of this project*

Reviewer: 链接是 https://www.transifex.com/linuxdeepin/deepin-archive-manager/
```suggestion
- [Translate for your language on Transifex](https://www.transifex.com/linuxdeepin/deepin-archive-manager/)
```

## License

> License description here. The license name is suggested to use the same one as [SPDX license identifier](https://spdx.org/licenses). Following is an example:

Reviewer: ```suggestion
```

> Deepin Example is licensed under [GPL-3.0-or-later](LICENSE)

Reviewer: ```suggestion
Deepin Compressor is licensed under [GPL-3.0-or-later](LICENSE)
```
